charts/tiphys/templates/app.yaml

```diff
@@ -323,6 +327,10 @@ spec:
             - name: app
               image: {{ default $.Values.defaultImage $app.image | quote }}
               imagePullPolicy: Always
+              {{ - if $.Values.imagePullSecrets }}
```

Comment:
This seems wrong?

Reply:
why? we're checking if value is passed if not then we will not create the imagepullsecret.

Reply:
ah sorry you mean with values. Yeah its fixed